src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -32,6 +32,8 @@
     public static final Block blockConstructionTapeCorner = new BlockConstructionTapeCorner();
     public static final Block blockTimberFrame            = new BlockTimberFrame();
     public static final Block blockRack                   = new BlockMinecoloniesRack();
+    public static final Block blockWayPoint               = new BlockWaypoint();
+
 
     // Deactivated for now
     // public static final Block blockBarrel        = new BlockBarrel();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)